js/projects.js

```diff
@@ -46,7 +46,7 @@ ${singleProject.project.stack}</li>
                       <p class="card-title weight--400">
 ${singleProject.description.long}</p>
```

Comment:
-  As the last code reviewer said: 👇🏽 
- [ ] Please 🙏🏽  use scroll for the text in the mobile version to be similar to the Figma design.
> Your text is a bit longer than the original design. I suggest you to limit it or you can use overflow method.
 - [ ] Correct the position of the skill buttons to the Figma design.
 > Your stacks (html, css, javaScript) are also really close to the description.
- [ ] Please 🙏🏽 correct the text inside the buttons.
 > As you can see in the video your buttons are also looks different.

| Figma design | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/209392299-1cbea88d-ea9a-4669-a325-eca58b5bff96.png) | ![image](https://user-images.githubusercontent.com/58642949/209392326-b898ec40-cf5b-4da3-8345-ab8becb23d50.png) |

Reply:
Hi, @franclobo 

every issue you listed here has been corrected before I made this PR, kindly check and know if u have mistakenly checked the wrong branch

Reply:
Hi, @boniface193 

I hope everything is going well. As you can see in the video and the picture that I saw the moment I was reviewing your project. I checked it now and it looks good, however, you haven't used the overflow method to limit the text in the mobile version to be similar to the Figma template, please correct it.

- **[COMPLETED]**  Correct the position of the skill buttons to the Figma design. ✔️ 
- **[OPTIONAL]** You corrected the text in the button, kindly correct the font size it is 15px instead of 12px.

Reply:
issues is resolved